js/models/conversations.js

```diff
@@ -198,10 +198,21 @@
     isOnline() {
       return this.isMe() || this.get('isOnline');
     },
-
     isMe() {
+      return this.isOurLocalDevice() || this.isOurPrimaryDevice();
+    },
+    isOurPrimaryDevice() {
       return this.id === window.storage.get('primaryDevicePubKey');
     },
+    async isOurDevice() {
+      const ourDevices = await window.libloki.storage.getPairedDevicesFor(
```

Comment:
minor optimization: we can avoid the call to the storage if we just check on this. isOurLocalDevice first
```
if (this.isOurLocalDevice()) {
  return true;
}
``` then proceed to load and check final

Reply:
Good catch, i can actually go further and just do `if (this.isMe()` and bail early if that's true